README.md

```diff
@@ -8,15 +8,22 @@ This project uses Poetry for dependency management.
 
 ```bash
 # Install dependencies
-poetry install
+poetry install --no-root
```

Comment:
did you try this:
'If you want to use Poetry only for dependency management but not for packaging, you can disable package mode by setting package-mode = false in your pyproject.toml file.'